src/__tests__/proposalVersions.int.test.ts

```diff
@@ -1059,8 +1059,8 @@ describe('/proposalVersions', () => {
           created_at
         )
         VALUES
-          ( 'First Name', 'firstName', 'string', '2022-07-20 12:00:00+0000' ),
-          ( 'Last Name', 'lastName', 'string', '2022-07-20 12:00:00+0000' );
+          ( 'First Name', 'firstName', '{ "type": "string" }', '2022-07-20 12:00:00+0000' ),
+          ( 'Last Name', 'lastName', '{ "type": "string" }', '2022-07-20 12:00:00+0000' );
```

Comment:
Can we use the `insertOne` SQL script for all of these? I believe `pg` knows how to convert JavaScript objects into JSONB data, so ideally we should be able to

```ts
db.sql('canonicalFields.insertOne', {
  label: 'First Name',
  shortCode: 'firstName',
  dataType: { type: 'string' },
});
```

I think it's worth splitting out a commit just for switching all these direct SQL calls into uses of existing scripts!